app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t(:base_title)
```

Comment:
convention: chỗ nào có thể lược bỏ được ngoặc tròn thì em sẽ bỏ đi nhé
```suggestion
    base_title = t :base_title
```